bundle-workflow/scripts/bundle-build/components/OpenSearch/build.sh

```diff
@@ -91,3 +91,16 @@ ARTIFACT_BUILD_NAME=opensearch-$VERSION$IDENTIFIER-$QUALIFIER.tar.gz
 ARTIFACT_TARGET_NAME=opensearch-min-$VERSION$IDENTIFIER-$QUALIFIER.tar.gz
 mkdir -p "${outputDir}/bundle"
 cp distribution/archives/$TARGET/build/distributions/$ARTIFACT_BUILD_NAME "${outputDir}"/bundle/$ARTIFACT_TARGET_NAME
+
+
+# Build core plugins
+echo "Building core plugins..."
+mkdir -p "${outputDir}/core-plugins"
+cd plugins
+../gradlew assemble -Dbuild.snapshot=$SNAPSHOT
+for plugin in *; do
+  if [ -d "$plugin" ] && [ "examples" != "$plugin" ]; then
+    PLUGIN_ARTIFACT_BUILD_NAME="$plugin"-$VERSION$IDENTIFIER.zip
+    cp "$plugin"/build/distributions/"$PLUGIN_ARTIFACT_BUILD_NAME" "../${outputDir}"/core-plugins/"$PLUGIN_ARTIFACT_BUILD_NAME"
```

Comment:
This forces `outputDir` to always be a relative path and will break in unexpected ways. Instead, `cd ..` after assembling inside `plugins` and iterate over the output from the parent folder. This will remove the need for `..` relative path here.

Reply:
Thanks :) Fixed.